compute/instances/reference-content/choosing-instance-type.mdx

```diff
@@ -28,6 +28,10 @@ Instances are virtual machines: multiple Instances can run on one "real" physica
         - **Shared**: A [context-switching mechanism](https://en.wikipedia.org/wiki/Context_switch) allows a physical core to be shared between multiple vCPUs for multitasking. Instances with shared vCPU can be more cost-efective for running small, non-resource intensive applications than dedicated vCPU Instances. Our different types of shared vCPU Instances offer a variety of options to balance price and performance depending on your use case.
         - **Dedicated**: Instances with dedicated vCPU do not share their compute resources with other Instances (1 vCPU = 1 CPU thread dedicated to that Instance). This type of Instance is particularly recommended for running production-grade compute-intensive applications
 
+        <Message type="tip">
+          Scaleway provides Instances equipped with both x86 and ARM CPUs. ARM Instances generally offer higher energy and cost efficiency, while x86 cloud instances provide wider software compatibility and stronger single-thread performance. [Learn more about the differences between ARM and x86 based Instances](compute/instances/reference-content/understanding-differences-x86-arm).
```

Comment:
> Instances equipped with both x86 and ARM CPUs.

This wording gives the impression of a single instance with both architectures at the same time. This could be reworded as follows:

> Instances equipped with either x86 or ARM CPUs.

Reply:
```suggestion
          Scaleway provides Instances equipped with either x86 and ARM CPUs. ARM Instances generally offer higher energy and cost efficiency, while x86 cloud instances provide wider software compatibility and stronger single-thread performance. [Learn more about the differences between ARM and x86 based Instances](compute/instances/reference-content/understanding-differences-x86-arm).
```